generators/app/templates/styles/itcss/components/_components.buttons.scss

```diff
@@ -0,0 +1,60 @@
+/* ==========================================================================
+   #BUTTONS
```

Comment:
Maybe let's change this to `components/_btn.scss`?
I like having the component name corresponding with its filename